boot/zephyr/main.c

```diff
@@ -42,6 +42,35 @@
 #include "bootutil/mcuboot_status.h"
 #include "flash_map_backend/flash_map_backend.h"
 
+/* Check if Espressif target is supported */
+#if defined(CONFIG_SOC_ESP32) || \
```

Comment:
What if you add a new kconfig at https://github.com/mcu-tools/mcuboot/blob/main/boot/zephyr/Kconfig defining 
```c
config ESPRESSIF_SOC
    depends on CONFIG_SOC_ESP32 || (...)
```